script/libtest.sh

```diff
@@ -58,3 +40,35 @@ k8s_ensure_image() {
         return 1
     fi
 }
+
+# Waits for a set of jobs matching the provided tag to be Completed.
+# It retries up to $TEST_MAX_WAIT_SEC
+k8s_wait_for_job_completed() {
+    namespace=${1:?}
+    labelSelector=${2:?}
+
+    local -i retryTimeSeconds=${TEST_MAX_WAIT_SEC:?}
+    local -i retryTimeStepSeconds=5
+
+    echo "Wait for job completion started"
+
+    set +e
```

Comment:
Could you explain why you needed to override the `set +e`?

Also, could it wrap just the `res=kubectl` stanza?